### PR TITLE
Added CLI config-filename support and error on mapping undefined field from csv

### PR DESCRIPTION
**Pull request by rozetko**
Saw this functionality in your issues. Actually, don't see now. Is it still needed?
Also, fixed some typos.

### CODE REVIEW

**Comment by rozetko** [comment]
Added error on mapping undefined field from csv
https://github.com/CorpGlory/csv2influx/issues/7

File: README.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ npm install -g csv2influx
 ## Usage
 
 ```
-csv2influx init        Creates template config file
-csv2influx data.csv    Loads config file from current directory
-                       then imports file data.csv to your influx
+csv2influx init                                       Creates template config file
+csv2influx --config path/to/config.json data.csv      Loads config from path/to/config.json
```
Comment (jonyrock) [inline README.md]: csv2influx.conf.json 
I don't want to be forced to put `--config` 
for example, when you use `npm install` , then npm tries to use `package.json` from current directory. Same thing here. On top of this, it breaks our working example. 

Btw, we call it `csv2influx.conf.json` , but not `config.json`

But he idea is cool.

File: index.js [hunk 2]
```diff
@@ -5,12 +5,11 @@ const importer = require('./importer');
 
 const usage = `
 Usage:
-  csv2influx init        Creates template config file
-  csv2influx data.csv    Loads config file from current directory
-                         then imports file data.csv to your influx
+  csv2influx init                                       Creates template config file
+  csv2influx --config path/to/config.json data.csv      Loads config from path/to/config.json then imports file data.csv to your influx
```
Comment (jonyrock) [inline index.js]: I proposed to make `--config` optional, so here would be great to add few words to help about it.

File: importer.js [hunk 3]
```diff
@@ -78,7 +78,19 @@ class Importer {
       stripUnknown: true,
     });
 
-    this.config.csv.columns = true;
+    this.config.csv.columns = (cols) => { // callback for checking columns names in csv
+      Object.keys(this.config.mapping.fieldSchema).forEach((key) => {
+        if (cols.indexOf(key) < 0) // if key doesn't exist in cols array
+        {
+          console.error('Error: there is no column named ' + key + ' in ' + inputFile);
+          console.log('column names: ' + cols)
```
Comment (jonyrock) [inline importer.js]: It's strange that you write something to `console.error` and then to `console.log` and then finish the process. 
I believe that you wanted to write to `console.error` again

**Comment by rozetko** [comment]
Fixed

P.S.
> Btw, we call it csv2influx.conf.json , but not config.json


Actually, it's not config.json but path/to/config.json. Seems to me like a standard way to indicate filename

**Review by jonyrock — APPROVED** [review]

**Pull request merged**